tests/system/test_system.py

```diff
@@ -922,6 +923,7 @@ def test_collection_group_queries_filters(client, cleanup):
     assert found == set(["cg-doc2"])
 
 
+@unittest.skipIf(FIRESTORE_EMULATOR, "PartitionQuery not implemented in emulator")
```

Comment:
Likely we would prefer using [`@pytest.mark.skipif`](https://docs.pytest.org/en/latest/how-to/skipping.html#id1):
```suggestion
@pytest.mark.skipif(FIRESTORE_EMULATOR, reason="PartitionQuery not implemented in emulator")
```